cfgov/ask_cfpb/models/pages.py

```diff
@@ -33,6 +35,32 @@
 ENGLISH_DISCLAIMER_SNIPPET_TITLE = 'Legal disclaimer for consumer materials'
 SPANISH_DISCLAIMER_SNIPPET_TITLE = (
     'Legal disclaimer for consumer materials (in Spanish)')
+CONSUMER_TOOLS_PORTAL_PAGES = {
```

Comment:
I wish there were a way to programmatically define portal page urls and slugs, rather than relying on a hard-coded list that would need to be updated.